packages/fx-core/src/common/azure-hosting/azureFunctionHosting.ts

```diff
@@ -18,4 +22,16 @@ export class AzureFunctionHosting extends AzureHosting {
     functionEndpoint: functionEndpoint,
     endpointAsParam: endpointAsParam,
   };
+
+  async deploy(
+    inputs: Inputs,
+    tokenProvider: TokenProvider,
+    buffer: Buffer,
+    siteName: string
+  ): Promise<Void> {
+    await super.deploy(inputs, tokenProvider, buffer, siteName);
+    const client = await azureWebSiteDeploy(inputs, tokenProvider, buffer, siteName);
+    await AzureOperations.RestartWebApp(client, inputs.resourceGroupName, siteName);
```

Comment:
I have a question about why the method name is not "restartWebApp" which is lower case?